open-codegen/opengen/builder/ros_builder.py

```diff
@@ -106,6 +107,7 @@ def __copy__ros_files(self):
                 target_ros_dir, 'include', header_file_name))
         original_include_file = os.path.abspath(
             os.path.join(self.__target_dir(), header_file_name))
+        print(original_include_file)
```

Comment:
might want to add a label to the print statement to say what the file is

Reply:
This print statement shouldn't be there. I've now removed it.